### ISSUE TITLE
Any normal user can remove anyone's uploads using delagent from the command line

### ISSUE THREAD

**Issue by yaobinshi** [fossology/fossology#186]
---

Author Name: **Mary Laser** 
Original Redmine Issue: 3063, http://www.fossology.org/issues/3063
Original Date: 2012/06/25
Original Assignee: Mary Laser

---

A new manualtest is needed to verify #1293 before it can be closed

**Comment by yaobinshi** [comment]
---

Original Redmine Comment
Author Name: **Mary Laser** 
Original Date: 2012/06/26 19:07:31 +0000

---

fixed title/description

**Comment by yaobinshi** [comment]
---

Original Redmine Comment
Author Name: **Bob Gobeille** 
Original Date: 2012/10/08 18:35:32 +0000

---

Change unranked issues to rank 99 so they show up in a ranking report at the bottom.  Otherwise, they show up on top.

**Comment by steffen-weber** [comment]
Please give a description of the referred issue or close this ticket.

**Comment by bobgob** [comment]
Here is the original problem referred to by the old 1293:

Any normal user can remove anyone's uploads and folders using delagent from the command line.

**Comment by mcjaeger** [comment]
should be closed by #646 would need retesting

**Comment by max-wittig** [comment]
If I understand the issue correctly. It is not fixed. 

What I did:
1. Uploaded files to FOSSology as 'fossy' upload user
2. Created a user test with `Read-only (read, but no writes or downloads)` Access Level
3. Ran `./delagent --user test --password 123 -U 142`
4. Result: <pre>NOTICE:  table "delup_142_pfile" does not exist, skipping
The upload '142' is deleted by the user 'test'. </pre>
5. Checked the FOSSology browse page as user fossy. The upload is gone.
